### ISSUE TITLE
evolving/maintenance roadmap

### ISSUE THREAD

**Issue by oleksmarkh** [obviel/obviel#8]
First off, I would like to thank you for a neat framework.
We use it mainly as a rendering/template + l10n engine.

My question is if there're any plans to put some love into this, giving it a change to survive our refactoring cycles :) If yes, I'll be happy to list our concerns in a form of issues here.

**Comment by faassen** [comment]
Thanks for asking, I'm glad it's working for you. Unfortunately I do not have plans to put energy into this; there are so many JS frameworks around these days. If someone else wants to take over maintenance I'm happy to give them access.

I myself have started using React, which is quite Obviel-like in its focus on small components, though of course its approach to templating is quite different.

**Comment by markhovskiy** [comment]
Thanks for answering! We also consider React these days. A good thing is that Obviel wasn't pretending to be a huge thing, doing it's rendering job nicely. We might just slice our view layer and rework it a bit - which is a great opportunity to recover. So again, thanks for that! If you don't mind, share this experience with @robgietema, I was pleased to work with him :)

And that's how it feels writing a framework today https://twitter.com/mpjme/status/602142966245883904

**Comment by faassen** [comment]
Yes, I do keep in touch with @robgietema ; we're in fact going to the React Europe conference together this summer.

Yeah, when I was first working on Obviel I figured I was creating something new: a client-side web framework. It turned out that I wasn't the only one with that idea -- the time was ripe for it, and Obviel is not one of the frameworks that emerged that got a lot of attention. It was however a positive experience: I learned a lot about JS web frameworks as a result. :)
